### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by greglandrum**
First of a series of PRs with improvements to the docs.
There are no actual code changes here.

A note on docstring formatting: when adding new docstrings for the python wrappers, I've started using the numpy/scipy format (https://numpydoc.readthedocs.io/en/latest/format.html) instead of whatever random thing we were doing before.
I'd suggeste that we adopt this format going forward and switch old docstrings over as time allows.

### CODE REVIEW

File: Code/GraphMol/MolOps.h [hunk 1]
```diff
@@ -1072,6 +1076,7 @@ RDKIT_GRAPHMOL_EXPORT void clearSingleBondDirFlags(ROMol &mol,
 //! removes directions from all bonds. Wiggly bonds and cross bonds will have
 //! the property _UnknownStereo set on them
 RDKIT_GRAPHMOL_EXPORT void clearAllBondDirFlags(ROMol &mol);
+//! removes directions from all bonds. Wiggly bonds and cross bonds will have
```
Comment (ricrogz) [inline Code/GraphMol/MolOps.h:1079]: will have ... ?

File: External/pubchem_shape/PubChemShape.hpp [hunk 2]
```diff
@@ -14,15 +15,52 @@ struct RDKIT_PUBCHEMSHAPE_EXPORT ShapeInput {
...
+/*!
+  \param ref          the reference molecule
+  \param fit          the molecule to align
+  \param matrix       the transformation matrix (populated on return)
+  \param refConfId    (optional) the conformer to use for the reference molecule
+  \param fitConfId    (optional) the conformer to use for the fit molecule
+  \param useColors    (optional) whether or not to use colors in the scoring
+  \param opt_param    (optional) the optimization parameter
+  \param max_preiters (optional) the max number of pre-optimization iterations
+  \param max_postiters(optional) the max number of post-optimization iterations
```
Comment (ricrogz) [inline External/pubchem_shape/PubChemShape.hpp:59]: Maybe a space separator before "(optional)" here\, for readability

File: Code/GraphMol/MolOps.h (hunk 1 above)
Comment (greglandrum) [reply Code/GraphMol/MolOps.h:1079]: will reword

**Comment by greglandrum** [comment]
Hi @ricrogz I made your suggested changes

**Review by ricrogz — APPROVED** [review]

**Pull request merged**